tools/editor/code_editor.cpp

```diff
@@ -1011,6 +1038,21 @@ void CodeTextEditor::_complete_request() {
 	text_editor->code_complete(strs);
 }
 
+void CodeTextEditor::_font_resize_timeout() {
+
+	Ref<DynamicFont> font = text_editor->get_font("font");
+
+	if (font.is_valid()) {
+		int size=font->get_size()+font_resize_val;
+		font_resize_val=0;
+
+		if (size>=8 && size<=96) {
```

Comment:
Can we use Ctrl+0 to reset the zoom? That's usually the standard

On 18 June 2016 at 12:18, Ignacio Etcheverry <notifications@github.com>
wrote:

> In tools/editor/code_editor.cpp
> <https://github.com/godotengine/godot/pull/5259#discussion_r67602247>:
>
> > @@ -1011,6 +1038,21 @@ void CodeTextEditor::_complete_request() {
> >  	text_editor->code_complete(strs);
> >  }
> >
> > +void CodeTextEditor::_font_resize_timeout() {
> > +
> > +	Ref<DynamicFont> font = text_editor->get_font("font");
> > +
> > +	if (font.is_valid()) {
> > +		int size=font->get_size()+font_resize_val;
> > +		font_resize_val=0;
> > +
> > +		if (size>=8 && size<=96) {
>
> @reduz <https://github.com/reduz> Still wondering if there is a way to
> get the property range limits
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/godotengine/godot/pull/5259/files/1f3de1fc02f87317afa06ba267c60fc0c339421c#r67602247>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGVmPY6rXvf0Zjx69VBk_UfsaZHoh5aYks5qNAxBgaJpZM4I4upG>
> .
>